scripts/run_nodes.sh

```diff
@@ -227,10 +228,11 @@ if ! killall -9 aleph-node 2> /dev/null; then
 fi
 
 if [[ -z "${DONT_BUILD_ALEPH_NODE}" ]]; then
-  info "Building testing aleph-node binary (short session)."
-  cargo build --release -p aleph-node --features "short_session enable_treasury_proposals"
-elif [[ ! -x "${ALEPH_NODE}" ]]; then
-  error "${ALEPH_NODE} does not exist or it's not an executable file!"
+  info "Building testing aleph-node binary (short session) and chain-bootstrapper binary."
+  cargo build --release -p aleph-node
+  cargo build --release -p chain-bootstrapper --features "short_session enable_treasury_proposals"
```

Comment:
NIt: chain-bootstrapper doesn't have to be built when `DONT_BOOTSTRAP`

Reply:
Good point